src/explorer/FolderTreeItem.ts

```diff
@@ -40,7 +40,8 @@ export class FolderTreeItem implements IAzureParentTreeItem {
         const kuduClient: KuduClient = await this.siteWrapper.getKuduClient(webAppClient);
         const httpResponse: kuduIncomingMessage = <kuduIncomingMessage>(await kuduClient.vfs.getItemWithHttpOperationResponse(this.folderPath)).response;
         // response contains a body with a JSON parseable string
-        const fileList = JSON.parse(httpResponse.body);
+        // tslint:disable-next-line:no-unsafe-any
+        const fileList: kuduFile[] = JSON.parse(httpResponse.body);

```

Comment:
nit: I prefer this rather than disabling the rule:
```
const fileList: kuduFile[] = JSON.parse(<string>httpResponse.body);
```

Reply:
tslint is still yelling at me :(

This fixes it though:
```
<kuduFile[]>JSON.parse(httpResponse.body)
```

I think it's because the return type of JSON.parse() is any which was causing the warning.